clifford/_mvarray.py

```diff
@@ -1,6 +1,10 @@
 import numpy as np
 
+import operator
+import functools
 
+from typing import Tuple
+from clifford import Layout
 from clifford.io import write_ga_file, read_ga_file  # noqa: F401
 from ._multivector import MultiVector
```

Comment:
```suggestion
from clifford.io import write_ga_file, read_ga_file  # noqa: F401
from ._layout import Layout
from ._multivector import MultiVector
```
makes it easier to reason about circular imports